File: ruby/easy/length_of_last_word.rb
# @param {String} s
# @return {Integer}
def length_of_last_word(s)
    i = s.length-1
    count = 0
    while i >= 0 && s[i] == ' '
        i -= 1
    end
    while i >= 0 and s[i] != ' '
        count += 1
        i -= 1
    end
    return count
end

File: javascript/easy/lengthOfLastWord.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let count = 0, i = s.length-1;
    while(i >= 0 && s[i] === ' ') i--;
    while(i >= 0 && s[i] !== ' ') { count++; i--; }
    return count;
};
